### ISSUE TITLE
Feature request: Add ability to delete decryption history

### ISSUE THREAD

**Issue by ghost** [pgptool/pgptool#235]
"Delete history and Key algorithm" Pretty self explanatory, let us delete history  and choose whether we want sha512 keys or sha256 keys. 
Thank you in advance and keep up the great work!!

**Comment by skarpushin** [comment]
@frztgi good idea, thanks, I've added history deletion as Feature request. As of now, if you want to delete this history, you can just delete file `ConfigPairsEnvelop.decr-params` while application is not running. This file is located in `/home/%username%/.pgptool/configs/`. On Windows replace `/home` with `C:\Users`

As for choosing key algorithm this is a planned feature #196

**Comment by frztgi** [comment]
So glad to see this project being updated. Thank you for making this awesome tool even better.
